controllers/content.js

```diff
@@ -9,35 +9,35 @@ const api = require('../api');
 // secure routes
 router.use(authHelper.authChecker);
 
-const contentFilterSettings = () => [
+const contentFilterSettings = (res) => [
 	{
 		type: 'sort',
-		title: 'Sortierung',
-		displayTemplate: 'Sortieren nach: %1',
+		title: res.$t('content.headline.sortBy'),
+		displayTemplate: res.$t('content.label.sortBy'),
```

Comment:
Does it work with %1? Have you tried that?

Reply:
![sortby%1](https://user-images.githubusercontent.com/45626086/84263844-819bdc80-ab20-11ea-834f-f5b0440d6e17.png)
works :)